analysis/dataset_definition_prevax.py

```diff
@@ -0,0 +1,22 @@
+from dataset_definition_cohorts import generate_dataset
+
+from ehrql.query_language import table_from_file, PatientFrame, Series
+
+from datetime import date
+
+# extract index dates for prevax cohort from index_dates.csv
+
+@table_from_file("output/index_dates.csv.gz")
+
+class index_dates(PatientFrame):
+    index_prevax = Series(date)
+    end_prevax_exposure = Series(date)
+    end_prevax_outcome = Series(date)
+
+index_date = index_dates.index_prevax
+end_date_exp = index_dates.end_prevax_exposure
```

Comment:
I think these need to be `end_date_exposure` and `end_date_outcome` as they are used throughout the rest of the code base like this. [This will apply to vax and unvax too.]